kolibri/plugins/style_guide/templates/style_guide/style_guide.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
unfortunately we can't introduce dependencies on the internet

Search our code for `mat-svg` to see an alternate way of including material design icons